packages/studio-web/extract-i18n-lang.ts

```diff
@@ -0,0 +1,57 @@
+// Update the language files with the new keys from the English file
+// Usage:
+//   npx tsx extract-i18n-lang.ts src/i18n/messages.json src/i18n/messages.fr.json > src/i18n/messages.fr-updated.json
```

Comment:
The README should be update too: https://github.com/ReadAlongs/Studio-Web?tab=readme-ov-file#internationalization-i18n-and-localization-l10n

Reply:
Good point.

Reply:
done